pkgs/os-specific/linux/afuse/default.nix

```
@@ -11,11 +11,18 @@ stdenv.mkDerivation {
   nativeBuildInputs = [ autoreconfHook pkg-config ];
   buildInputs = [ fuse ];
 
+  postPatch = ''
```

```suggestion
  postPatch = lib.optionalString stdenv.isDarwin ''
```
if it's specific to darwin

---

Fixed.